articles/contributors.re

```diff
@@ -1,16 +1,21 @@
 = 著者/デザイナー紹介
-=== Goとベイズ理論でゼロからシンプルな記事分類を実装してみよう！ / @po3rin
+==== GUIライブラリGioで遊んでみよう / @knsh14
+メルペイでバックエンドエンジニアとして決済サービスを運用している。
+GopherConで英語のできなさに打ちひしがれたので、頑張って勉強することを決意した。
+まずは好きなサッカーを英語実況で楽しむところから始めます。
+
+==== Goとベイズ理論でゼロからシンプルな記事分類を実装してみよう！ / @po3rin
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)